upgrade/UPGRADE-v8.2.0-dev.md

```diff
@@ -1,6 +1,6 @@
-# [Upgrade from v8.1.0 to v8.2.0-dev](https://github.com/shopsys/shopsys/compare/v8.1.0...master)
+# [Upgrade from v8.1.1 to v8.2.0-dev](https://github.com/shopsys/shopsys/compare/v8.1.1...8.1)
```

Comment:
As we don't plan to release v8.2.0, I suggest to delete this file completely and links to this file. 
Please validate this idea with @sslt 

Or is necessary to change link (v8.1.1...8.1 is not right)

Reply:
Removed